src/plugins/data_views/common/fields/data_view_field.ts

```diff
@@ -298,6 +298,10 @@ export class DataViewField implements DataViewFieldBase {
    * Returns true if field is sortable
    */
   public get sortable() {
+    if (this.scripted && this.spec.type === 'date') {
```

Comment:
Thx, approved, field is no longer sortable, also in the UI 👍 
![Bildschirmfoto 2024-05-28 um 12 05 46](https://github.com/elastic/kibana/assets/463851/1a0e9ba4-66bc-4ef4-8052-67dc3e2a4df8)